src/io/output/WritePVD.cpp

```diff
@@ -3,7 +3,7 @@
 static void strip_ext(char *);
 
 // Write the pvtu file if you are rank zero and code in parallel
-void WritePVD(const char* FileName, int step,double time){
+void WritePVD(const char* FileName, int step, double* stepTime){
```

Comment:
@eralpsahin if stepTime is global does it need to be sent into array? @enstrophy

Reply:
You need not pass the array.